README.md

```diff
@@ -27,6 +27,12 @@ A complete event-driven application that includes everything you need to build a
 
 Let's jump in and get the event-driven app up and running in Azure. When you are finished, you will have a fully functional event driven app deployed on Azure. In later steps, you'll see how to setup a pipeline and run the application.
 
+| Deploy to Azure Spring Apps | |
```

Comment:
Maybe we should add some description on top of this table. 

"""
Before delving into the step-by-step execution of the application, you can simply click the Deploy to Azure button. This will instantly deploy the app to Azure Spring Apps.
"""

And this sentence should be put above the line 28.